examples/contracts/examples/contract_deploy_anvil.rs

```diff
@@ -1,10 +1,11 @@
-use ethers::{
-    prelude::*,
-    solc::{Project, ProjectPathsConfig},
-    utils::Anvil,
-};
+use ethers_contract::{abigen, ContractFactory};
```

Comment:
We should communicate clearly that while these are the minimal crates, the ethers crate re exports everything and is easier to use unless you're trimming dependencies

Reply:
I updated all manifests to reference the "ethers" crate. All imports follow the `ethers::` standard.